### PR TITLE
BOAC-2216: Uses pre-built student profile summary feeds

### PR DESCRIPTION
**Pull request by lyttam**
https://jira-secure.berkeley.edu/browse/BOAC-2216

More work to follow: add enrollments, term GPA, and academic standing to profiles prior to stashing them in RDS.

### CODE REVIEW

**Comment by lyttam** [comment]
@pauline2k @johncrossman Initial build failure was of the occasional randomly-generated name variety. I re-ran it and it passed.

**Comment by johncrossman** [comment]
Thanks @lyttam

File: boac/merged/student.py [hunk 1]
```diff
@@ -224,14 +216,35 @@ def get_distinct_sids(sids=(), cohort_ids=(), curated_group_ids=()):
...
 
-def get_summary_student_profiles(sids, term_id=None):
+def get_student_profile_summaries(sids, term_id=None):
     if not sids:
         return []
-    benchmark = get_benchmarker('get_summary_student_profiles')
+    benchmark = get_benchmarker('get_student_profile_summaries')
     benchmark('begin')
-    # TODO It's probably more efficient to store summary profiles in the loch, rather than distilling them
-    # on the fly from full profiles.
```
Comment (pauline2k) [inline boac/merged/student.py:233]: That's a nice longstanding TODO to get rid of.

File: fixtures/loch/student_profile_3456789012.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "canvasUserId": "10000002",
-    "canvasUserName": "Paul Kerschen",
-    "firstName": "Paul",
+    "canvasUserName": "Pauline Kerschen",
+    "firstName": "Pauline",
```
Comment (pauline2k) [inline fixtures/loch/student_profile_3456789012.json:4]: 🙏

**Pull request merged**